src/Carousel.js

```diff
@@ -12,15 +12,29 @@ class Carousel extends React.Component {
   componentDidMount() {
     const { options = {} } = this.props;
 
+    if (typeof window.$ !== 'undefined') {
+      window.$(this._carousel).carousel(options);
+    } else if (typeof window.M !== 'undefined') {
```

Comment:
let's just call it `M` as per the other implementations

Reply:
I'm having problem with this one. It doesn't work if I use "M" or "$" directly without using "window" before. I suspect it might be a problem of mine, though.
What should I do?

Reply:
Are you having problems in the specs or when running locally?

Reply:
I'm having problem running the tests.
And, if I try to compile the library and use it in a project of mine it will not work as well.
The same problem I'm having on #650

Reply:
Try it out on this repo: git@github.com:react-materialize/webpack-boilplate.git
Works fine for me :)